Tutorials/Editors-Manual/Getting-Started-With-Umbraco/index-v7.md

```diff
@@ -0,0 +1,43 @@
+---
+versionFrom: 7.0.0
+---
+
+# Getting started with Umbraco
+
+## [Logging In and Out](Logging-In-and-Out)
+
+## [Umbraco Interface](Umbraco-Interface/index-v7.md)
+
+### [Initial View](Umbraco-Interface/index-v7.md/#initial-view)
+
+### [Various Screen Sizes](Umbraco-Interface/index-v7.md/#various-screen-sizes)
```

Comment:
"Various screen sizes" isn't a heading on the Umbraco Interface article 🤔 
Perhaps it could be removed? Or chance to "Behaviour at Smaller Screen Sizes" which is the heading used instead?
What do you think?

Reply:
i did remove it 🤔